athena-federation-sdk/src/main/resources/com/amazonaws/athena/connector/lambda/data/zone-index.properties

```diff
@@ -0,0 +1,2237 @@
+#
+# DO NOT REMOVE OR MODIFY EXISTING ENTRIES
+#
+# This file contain the fixed numeric id of every supported time zone id.
+# Every zone id in this file must be supported by java.util.TimeZone and the
+# Joda time library.  This is because Presto uses both java.util.TimeZone and
+# the Joda time## for during execution.
+#
+# suppress inspection "UnusedProperty" for whole file
```

Comment:
This file makes me super nervous. Can't we get this from Java runtime somehow? what happens if we country changes its offset?